runtime/src/main/java/org/corfudb/protocols/wireprotocol/CorfuMsgType.java

```diff
@@ -31,42 +31,6 @@
     // Layout Messages
     LAYOUT_NOBOOTSTRAP(19, TypeToken.of(CorfuMsg.class), true, false),
 
```

Comment:
It might not worth the effort to remove this type at this time, might be better to remove when the LR is supported in Protobuf (i.e. at the time we don't need CorfuMsg at all)